portal/messaging/messages.php

```diff
@@ -20,25 +20,25 @@
 // Need access to classes, so run autoloader now instead of in globals.php.
```

Comment:
for example this script can be used by patient portal and core (via their own session/cookies).